### ISSUE TITLE
umLet 14.1.0 not working with Eclipse Mars 4.5.2

### ISSUE THREAD

**Issue by mohdsyazani** [umlet/umlet#373]
Hi,

When loading umLet to Eclipse Mars, I placed the plugin under plugin directory and did the eclipse -clean all when restart. The symptom is that if I click New->Other (select Umlet), I see no respond to the click. Here's the error I saw in the Error Log (snippet).

Caused by: java.lang.ClassNotFoundException: org.eclipse.jdt.core.IJavaElement cannot be found by com.umlet.plugin_14.1.0
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:439)

Regrds,

**Comment by mohdsyazani** [comment]
If it helps, I have java version "1.8.0_77" installed on my machine.

**Comment by afdia** [comment]
This was an error with v13.3 but it has been fixed in v14.1 (#331)
I've tried to reproduce it by downloading Eclipse C/C++ from https://www.eclipse.org/downloads/ and adding umlet to the dropins directory.
I can create new diagrams and open existing ones, so I guess it should work.

I would have guessed that you use an old version but the error log says 14.1.0
Perhaps check the MANIFEST.MF as described in #331 to make sure the dependency to org.eclipse.jdt.core is optional

Other than that please try to re-download Eclipse and the Umlet 14.1 plugin and use a fresh workspace to see if the problem persists

**Comment by mohdsyazani** [comment]
Thank you for the response. I looked at the manifest and looks like the dependency has already been set to optional for this release. After further digging, I then realized that my eclipse C/C++ does not have the Eclipse JDT plugin. I installed it via Help, and that fixes it.

**Comment by afdia** [comment]
It's good to know that you have found a workaround, but It's still weird that it doesn't work without JDT in your case.

Do you have any other Eclipse Plugins installed which could probably cause these stricter dependencies, or have you tried it with a fresh Eclipse C/C++ installation and Umlet 14.1 plugin?

**Comment by mohdsyazani** [comment]
Yes I did try another machine that is download both from the website and notice the same problem. I didn't have other plugins installed, Umlet is the first one. Yes its kind of odd that it only works for me when I have JDT installed. What gave me the clue was when I looked at the Eclipse View log error, it was looking for IJavaElement.

**Comment by afdia** [comment]
Ok thanks, I'll leave this issue open for some time to see if anyone else has similar problems.
Could you post your OS please? Perhaps this is also a factor for this issue.

Thanks

**Comment by mohdsyazani** [comment]
Noted sounds good. I have this running on Microsoft Windows 7.

Regards,
